packages/apollo-client/src/core/QueryManager.ts

```diff
@@ -1167,6 +1167,9 @@ export class QueryManager<TStore> {
     const { clientQuery } = this.transform(query);
     if (clientQuery) {
       observable = asyncMap(observable, result => {
+        if (result.errors) {
```

Comment:
```suggestion
        if (graphQLResultHasError(result)) {
```
This checks `result.errors.length` as well, and minifies slightly better.